deploy/kubernetes/rbac.yaml

```diff
@@ -83,14 +83,6 @@ metadata:
   namespace: default
   name: external-provisioner-cfg
 rules:
-# Only one of the following rules for endpoints or leases is required based on
-# what is set for `--leader-election-type`. Endpoints are deprecated in favor of Leases.
-- apiGroups: [""]
-  resources: ["endpoints"]
-  verbs: ["get", "watch", "list", "delete", "update", "create"]
-- apiGroups: ["coordination.k8s.io"]
-  resources: ["leases"]
-  verbs: ["get", "watch", "list", "delete", "update", "create"]
```

Comment:
The PR description only talks about "Considering we have removed leader election based on endpoints". Then why are the RBAC rules for leases also getting removed?

Reply:
adjusted .. thanks.